frame/ethereum/src/lib.rs

```diff
@@ -421,7 +421,7 @@ impl<T: Config> Pallet<T> {
 					(d.logs.clone(), d.used_gas)
 				}
 			};
-			cumulative_gas_used = used_gas;
+			cumulative_gas_used += used_gas;
```

Comment:
Each receipt already encodes the cumulative gas used, so the sum is not needed here (`block_cumulative_gas == last_receipt_cumulative_gas`).

Reply:
Thanks. I understand now.